app/src/styles.sass

```diff
@@ -152,18 +154,26 @@ $itero-mat-theme: mat-light-theme(( color: ( primary: $itero-mat-primary, accent
 @include angular-material-typography($itero-mat-typography)
 
 
-.sidemenu
-  border-left:1px solid #704f6a
-  border-top: 1px solid #704f6a
-  border-bottom: 1px solid #704f6a
+
+.info-panel
+  border: 1px solid #704f6a
+  border-right: none
   background-color: #d5b6d0
-  float: right
-  height: 100%
-  margin-right: -16px
-  margin-left: 3px
-  box-shadow: -2px 0px 2px 0 rgba(255, 204, 244, 0.9)
-  .helpbutton
-    text-align: center
+  box-shadow: -5px 0px 5px 0 rgba(255, 204, 244, 0.7)
+  width: 80%
```

Comment:
Please move that line to line 168, such that it is closer to its modification when the screen is wider.

Reply:
But when I delete line 163-168, it will be look like this 
![Capture d’écran 2021-06-10 à 10 56 51](https://user-images.githubusercontent.com/83946943/121496252-92ed2280-c9da-11eb-9e5e-ab79026dc114.png)

Reply:
Yes. Please do **not** remove any of these lines. Just move line 163 such that is is just before the line with @media.

Reply:
Ah okay I get it, just to change the position.